oasst-shared/oasst_shared/schemas/inference.py

```diff
@@ -8,20 +8,15 @@
 import psutil
 import pydantic
 import pynvml
+from oasst_shared.shared_settings import shared_settings
 
 INFERENCE_PROTOCOL_VERSION = "1"
-DEFAULT_MODEL_NAME = "distilgpt2"
 
 DEFAULT_MODEL_LENGTHS = {
     "_lorem": 1024,
-    "gpt2": 1024,
-    "distilgpt2": 1024,
-    "OpenAssistant/oasst-sft-1-pythia6b": 2048,
     "OpenAssistant/oasst-sft-1-pythia-12b": 2048,
     "OpenAssistant/oasst_sft_llama_7b_mask_1000": 2048,
     "OpenAssistant/oasst_sft_llama_13b_mask_1500": 2048,
```

Comment:
We can't offer these LLaMa models now either, right?

Reply:
yes we can, as long as we don't profit monetarily. we can even provide the outputs without providing the actual weights until we figure out the license stuff

Reply:
Ok - is the reason for removing 30B just that it would require too much compute then?

Reply:
yes exactly